scopes/semantics/schema/schema.main.runtime.ts

```diff
@@ -94,6 +94,7 @@ export class SchemaMain {
     // if on workspace get schema from ts server
     if (this.workspace) {
       const env = this.envs.getEnv(component).env;
+      const devPatterns = env.getDevPatterns?.(component);
```

Comment:
I think you should use 
https://github.com/teambit/bit/blob/8401bdfb8f56a7a57508c3407a21fea4fea635f0/scopes/component/dev-files/dev-files.main.runtime.ts#L185 
instead

Reply:
reply test